Src/AutoFixture.xUnit.net2/AutoDataDiscoverer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    internal class AutoDataDiscoverer : DataDiscoverer
```

Comment:
@ploeh it does work; before this change, visual studio gave errors in the output window about mismatched test names for those scenario tests using the auto attributes. After this change, tests did not have data in their names, and no output errors occurred.

My concern is that we might be relying on undocumented or accidental behaviour.